react-components/src/components/Reveal3DResources/queryMappedData.ts

```diff
@@ -91,26 +86,25 @@ async function getAncestorNodesForTreeIndex(
 
 async function getMappingEdges(
   fdmClient: FdmSDK,
-  fdmConfig: FdmAssetMappingsConfig,
   model: CogniteCadModel,
   ancestorIds: CogniteInternalId[]
-): Promise<{ edges: Array<EdgeItem<Record<string, any>>> }> {
+): Promise<{ edges: Array<EdgeItem<InModel3dEdgeProperties>> }> {
   const filter = {
     and: [
       {
         equals: {
           property: ['edge', 'endNode'],
           value: {
-            space: fdmConfig.global3dSpace,
-            externalId: `model_3d_${model.modelId}`
+            space: INSTANCE_SPACE_3D_DATA,
+            externalId: `${model.modelId}`
```

Comment:
Have we adopted this new convention in naming 3D models in FDM? Sounds good to me

Reply:
Backend has changed the convention, yes.